src/main/java/com/rpg2014/wrappers/MinecraftDynamoWrapper.java

```diff
@@ -31,100 +29,88 @@ private MinecraftDynamoWrapper() {
 
     public boolean isServerRunning() {
         try {
-            key = new HashMap<>();
-            return getItem(key, SERVER_RUNNING).get(VALUE).bool();
+            return getItem(SERVER_RUNNING).get(VALUE).bool();
         } catch (DynamoDbException e) {
             throw new InternalServerErrorException("Failed to get isServerRunning");
         }
     }
 
     public void setServerRunning() {
         try {
-            key = new HashMap<>();
-            updatedValues = new HashMap<>();
-            setItem(key, updatedValues, SERVER_RUNNING, true);
+            setItem(SERVER_RUNNING, true);
         } catch (DynamoDbException e) {
```

Comment:
Is it possible to move the try catch block to the getItem and setItem methods?
Would it be worth moving?